jac-uk/digital-platform
# 1307 - activity logging

## What's included?
Log's events in the system each time the status or stage of an application is updated.

## Who should test?
✅ Product owner
✅ Developers

## How to test?

- Navigate to one of the Stages pages for an exercise and change the status/stage of one or more applications.
- Click on the events link (top of page).
- Events should have been logged.

Note: The Events link at the top of the page will not be available in the Admin portal until #1246 - Events (display logged events) has been merged into develop. 

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas. Specifically, any part of the system that changes the stage/status of an application.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
